org.opent2t.sample.hub.superpopular/com.insteon.hub/js/common.js

```diff
@@ -0,0 +1,22 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+
+'use strict';
+
+class accessTokenInfo {
```

Comment:
Wink, Nest, and Hue does not really use the common.js files in their hub translators. Should I remove the common.js in for those hub translator as well?